libbeat/dashboards/import_dashboards.go

```diff
@@ -448,16 +448,15 @@ func (imp Importer) ImportDir(dirType string, dir string) error {
 	// check if the directory exists
 	if _, err := os.Stat(dir); err != nil {
 		// nothing to import
-		fmt.Println("No directory", dir)
-		return nil
+		return fmt.Errorf("No directory %s", dir)
```

Comment:
I did some work on this in the meantime to clean up the error handling and outputting of status messages, so I pushed that. Even if this PR can't be merged as-is, it might be useful for someone else doing the change you talked about to move `import_dashboards`.